code/game/atoms_movable.dm

```diff
@@ -45,7 +45,6 @@
 	var/atom/movable/moving_from_pull
 	///Holds information about any movement loops currently running/waiting to run on the movable. Lazy, will be null if nothing's going on
 	var/datum/movement_packet/move_packet
-	var/datum/forced_movement/force_moving = null //handled soley by forced_movement.dm
```

Comment:
This was the real reason I made this PR to begin with - reducing the amount of crud we have to scroll through on view variables stuff